### PR TITLE
Guppy auto branch

### PR DESCRIPTION
**Pull request by ericgraf**

### CODE REVIEW

File: flux-manifests/cluster-api-provider-cloud-director.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@ api_version: generators.giantswarm.io/v1
 app_catalog: control-plane-catalog
 app_destination_namespace: giantswarm
 app_name: cluster-api-provider-cloud-director
-app_version: 0.4.1
+app_version: 0.5.0
```
Comment (erkanerol) [inline flux-manifests/cluster-api-provider-cloud-director.yaml:5]: https://github.com/giantswarm/cloud-director-app-collection/blob/main/flux-manifests/cluster-api-provider-cloud-director.yaml#L5

File: flux-manifests/dns-operator-route53.yaml [hunk 2]
```diff
@@ -2,7 +2,7 @@ api_version: generators.giantswarm.io/v1
 app_catalog: control-plane-catalog
 app_destination_namespace: giantswarm
 app_name: dns-operator-route53
-app_version: 0.7.0
+app_version: 0.7.3
```
Comment (erkanerol) [inline flux-manifests/dns-operator-route53.yaml:5]: https://github.com/giantswarm/cloud-director-app-collection/blob/main/flux-manifests/dns-operator-route53.yaml#L5

**Review by erkanerol — COMMENTED** [review]
Everything is up-to-date in main branch. We don't need this PR.

**Comment by glitchcrab** [comment]
guppy is dead

**Pull request closed without merging**